torch_geometric/nn/fx.py

```diff
@@ -261,11 +265,99 @@ def maybe_replace_node(n: Node) -> Node:
 def symbolic_trace(
         module: Module,
         concrete_args: Optional[Dict[str, Any]] = None) -> GraphModule:
+
+    from torch_geometric.nn import Aggregation
```

Comment:
```suggestion
    from torch_geometric.nn import Aggregation
    import torch.fx._symbolic_trace as st
```
How about importing it here?

Reply:
I don't like imports inside classes.. but it makes a lot of sense here. Thanks :-)